### DISCUSSION TITLE
get all the selected models in nova action

### DISCUSSION THREAD

**Discussion by khawala** [laravel/nova-issues#5966] [Q&A]
- Laravel Version: 10
- Nova Version: 4
- PHP Version: 8.1
- Database Driver & Version:mysql

hello all i need to get all the selected resource ids in nova action and then send them to route
but i'm unable to get all of these models 
i only get 200 items at most in don't know why?

**Comment by khawala** [comment]
any replay please

**Comment by m9tdev** [comment]
Found a workaround! See https://github.com/laravel/nova-issues/issues/1113
